src/Components/Organisms/FOHeader/index.tsx

```diff
@@ -1,111 +1,76 @@
-                  id="Fill-3"
-                  fill="#FFFFFE"
-                />
-                <polyline
-                  id="Fill-4"
-                  fill="#FFFFFE"
-                  points="410.496 16.146 432.167 16.052 432.541 100.796 410.87 100.89 410.496 16.146"
-                />
-                <polyline
-                  id="Fill-5"
-                  fill="#FFFFFE"
-                  points="471.172 79.682 502.043 79.547 502.137 100.49 443.192 100.748 443.117 84.041 472.813 36.819 443.515 36.947 443.423 16.001 500.8 15.751 500.874 32.453 471.172 79.682"
-                />
-                <path
-                  d="M533.149,74.505 C537.523,78.805 542.981,80.939 549.518,80.9132347 C556.056,80.884 561.494,78.701 565.835,74.363 C570.175,70.028 572.329,64.51 572.29833,57.811 C572.269,51.111 570.067,45.612 565.688,41.313 C561.312,37.017 555.856,34.881 549.319,34.9106884 C542.78,34.937 537.339,37.12 533.002,41.456 C528.661,45.793 526.507,51.312 526.535711,58.01 C526.568,64.71 528.77,70.209 533.149,74.505 Z M581.093,89.186 C572.533,97.74 562.039,102.044 549.612,102.100476 C537.183,102.152 526.652,97.94 518.02,89.463 C509.386,80.987 505.043,70.534 504.987511,58.105 C504.935,45.677 509.186,35.188 517.743,26.633 C526.303,18.081 536.795,13.777 549.224,13.7234683 C561.652,13.669 572.185,17.881 580.817,26.356 C589.45,34.834 593.793,45.29 593.846546,57.716 C593.902,70.147 589.651,80.634 581.093,89.186 L581.093,89.186 Z"
-                  id="Fill-6"
-                  fill="#FFFFFE"
-                />
-                <polyline
-                  id="Fill-7"
-                  fill="#FFFFFE"
-                  points="650.435 15.092 671.983 14.999 672.356 99.743 655.649 99.815 625.561 57.334 625.749 99.947 604.08 100.042 603.706 15.297 620.531 15.225 650.622 57.707 650.435 15.092"
-                />
-                <polyline
-                  id="Fill-8"
-                  fill="#D8232A"
-                  points="114.355 0.999 45.728 0.999 45.728 46.996 68.592 46.996 68.592 23.863 91.49 23.863 91.49 116 114.355 116 114.355 0.999"
-                />
-                <polyline
-                  id="Fill-9"
-                  fill="#D8232A"
-                  points="0 116 68.628 116 68.628 70.004 45.763 70.004 45.763 93.136 22.865 93.136 22.865 0.999 0 0.999 0 116"
-                />
-              </g>
-            </svg>
-            <div style={{ flexGrow: 1 }}></div>
-          </>
-        ) : (
-          <>
-            <FAIconButton
-              testID="icon-button-header"
-              size="large"
-              edge="start"
-              color="inherit"
-              aria-label="menu"
-              sx={{ mr: 2 }}
-            >
-              <MenuIcon />
-            </FAIconButton>
+                <MenuIcon />
+              </FAIconButton>
+            </>
+          )}
+          {logo && <Box sx={{ mr: 3 }}>{logo}</Box>}
```

Comment:
Please Create Atom Logo

Reply:
I think its better for the developer who choose the best logo they wanna use. but I made a Logo Component just in case.